### DISCUSSION TITLE
Would it be possible to integrate an Excalidraw diagram?

### DISCUSSION THREAD

**Discussion by ijpatricio** [slidevjs/slidev#738] [Ideas]
Using the hosted Excalidraw service, I think they are embeddable so with an iframe would be possible...

But as the component is open source (React), would it be much difficult to have an integrated version also?

**Comment by thiswallz** [comment]
Im used it within an electron app and also with iframes, works just fine (https://github.com/avastjs/livecode-sessions), I would not go maybe as components since you would need to take care of the infrastructure and that leads to a lot of time consumption just to avoid using an iframe. I believe you will be fine using as iframe, is not like they will remove this option.

**Comment by ijpatricio** [comment]
@thiswallz thanks

I guess the online demos from your repo need some care.

I guess I was overengineering, I'll stick to the iframe then. Internet connection needed but apart from that, everything will go smooth I guess!

Thanks

**Reply by thiswallz** [reply]
yes, Im just creating that demo, not finished :D , but you should be ok by using iframes !

**Comment by andymac4182** [comment]
This would be amazing. I use Obsidian.md to work on most of my notes / presentations and have local excalidraw files I would love to include in my presentations https://github.com/zsviczian/obsidian-excalidraw-plugin
